### PR TITLE
Clizify weighted blending CLI

### PR DESCRIPTION
**Pull request by cgsandford**
Testing:
 - [x] Ran tests and they passed OK

### CODE REVIEW

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4667721&bid=15623571)

File: lib/improver/cli/weighted_blending.py [hunk 1]
```diff
@@ -32,213 +32,66 @@
...
-import numpy as np
-
-from improver.argparser import ArgParser
-from improver.blending.calculate_weights_and_blend import WeightAndBlend
-from improver.utilities.cli_utilities import load_json_or_none
-from improver.utilities.load import load_cubelist
-from improver.utilities.save import save_netcdf
-
-
-def main(argv=None):
```
Comment (LaurenceBeard) [inline lib/improver/cli/weighted_blending.py]: Need to remove the reference to main()
```
if __name__ == "__main__":
    main()
```

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4667721&bid=15623571)

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/app/metoppv_tech/improver/pullRequest?prid=4667721&bid=15623571)

**Pull request merged**